app/code/Magento/Ui/view/base/web/templates/grid/columns/image-preview.html

```diff
@@ -0,0 +1,22 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<div class="masonry-image-preview" if="$col.isVisible($row())" data-image-preview ko-style="$col.getStyles($row())">
+    <div class="container">
+        <div class="action-buttons">
+            <button class="action-previous" type="button" click="function () { $col.prev($row()); }">
+                <span translate="'Previous'"/>
+            </button>
+            <button class="action-next" type="button" click="function () { $col.next($row()); }">
```

Comment:
re: the inline function: If we have to inline a function expression, it would probably be better to just move that code to the view model. That way we get things like linting still. Can expose it as `onNextClick` or something like that.

Reply:
or `click="$col.next.bind($col, $row())"`

Reply:
Changed